modules/gapi/include/opencv2/gapi/infer/onnx.hpp

```diff
@@ -11,6 +11,7 @@
 #include <string>
 #include <array>
 #include <tuple> // tuple, tuple_size
+#include <codecvt> // wstring_convert
```

Comment:
Please move it to `.cpp` if you only need it in `.cpp`.

Reply:
Tests use conversion to wstring on Windows.

Reply:
Then tests should include this, but not your generic public header.

Reply:
@alalek, Moved  `#include <codecvt>` from onnx.hpp